pkg/test/framework/components/istio/operator.go

```diff
@@ -562,6 +562,7 @@ func (i *operatorComponent) generateCommonInstallSettings(cfg Config, cluster re
 		"-f", defaultsIOPFile,
 		"-f", iopFile,
 		"--set", "values.global.imagePullPolicy=" + s.PullPolicy,
+		"--set", "values.pilot.env.ENABLE_ADMIN_ENDPOINTS=true",
```

Comment:
should probably be in the `defaultsIOPFile` if this is intended for merging and not just testing locally